src/components/spatial/Spatial.js

```diff
@@ -179,23 +254,26 @@ export default class Spatial extends AbstractSelectableComponent {
   }
 
   renderImageLayers() {
-    const layers = this.images.map(layer => this.createTileLayer(layer));
+    const layers = this.images.map(layer => this.createImageTileLayer(layer));
     return layers;
   }
 
-  createTileLayer(layer) {
+  createImageTileLayer(layer) {
     const [layerType, source] = layer;
     const minZoom = Math.floor(-1 * Math.log2(Math.max(source.height, source.width)));
     return new BaseTileLayer({
       id: `${layerType}-${source.tileSource}-tile-layer`,
       pickable: true,
       coordinateSystem: COORDINATE_SYSTEM.CARTESIAN,
       getTileData: ({ x, y, z }) => load(`${source.tileSource}/${layerType}_files/${z - minZoom}/${x}_${y}.jpeg`),
-      getTileIndices: (viewport, maxZoom, minZoom) => {
-        return getTileIndices(viewport, maxZoom, minZoom, source.tileSize, source.width, source.height)
+      // eslint-disable-next-line  arrow-body-style
```

Comment:
If there's some reason this is getting in your way, let's disable the rule, but I'd like to understand why.

Reply:
Let's talk offline....